packages/app/src/ui/pages/Popup/usePopup.ts

```diff
@@ -31,6 +32,7 @@ export const usePopup = (): IUsePopupData => {
   const url = new URL(window.location.href.replace("#", ""));
   const redirectParam = url.searchParams.get("redirect");
   const redirect = redirectParam && REDIRECT_PATHS[redirectParam];
+  const isCommonPath = useMemo(() => [Paths.RECOVER].includes(location.pathname as Paths), [location.pathname]);
```

Comment:
Is this condition temporary?

Reply:
No, we need it to show pages like recover without login or creating account.